app/assets/javascripts/discourse/tests/acceptance/users-test.js

```diff
@@ -37,6 +37,19 @@ acceptance("User Directory", function () {
     );
   });
 
+  test("Visit With Group Exclusion", async function (assert) {
```

Comment:
I'm not sure if I'm missing something but it doesn't seem like any of the assertions here are testing that groups are testing that groups are excluded?

Reply:
This is based on all the other tests of this file.
I already tested the backend on another PR.
I can improve this one if needed.